packages/cli/lib/index.ts

```diff
@@ -19,7 +19,7 @@ import { DryRunService } from './services/dryrun.service.js';
 import { v1toV2Migration, directoryMigration } from './services/migration.service.js';
 import { getNangoRootPath, upgradeAction, NANGO_INTEGRATIONS_LOCATION, printDebug } from './utils.js';
 import type { ENV, DeployOptions } from './types.js';
-import { load } from './services/config.service.js';
+import { loadValidateParse } from './services/config.service.js';
```

Comment:
I would simply call it parse. For me loading and validation is included in parsing

Reply:
fair